app/form_models/jobseekers/job_application/personal_details_form.rb

```diff
@@ -1,7 +1,21 @@
 class Jobseekers::JobApplication::PersonalDetailsForm
   include ActiveModel::Model
 
-  attr_accessor :first_name
+  attr_accessor :building_and_street, :email_address, :first_name, :last_name, :national_insurance_number,
+                :phone_number, :previous_names, :postcode, :teacher_reference_number, :town_or_city
 
-  validates :first_name, presence: true
+  EMAIL_FORMAT = /\A([^@\s]+)@((?:[-a-z0-9]+\.)+[a-z]{2,})\z/.freeze
+  PHONE_NUMBER_FORMAT = /\A\+?(?:\d\s?){10,12}\z/.freeze
+  NATIONAL_INSURANCE_NUMBER_FORMAT = /\A\s*[a-zA-Z]{2}(?:\s*\d\s*){6}[a-zA-Z]?\s*\z/.freeze
```

Comment:
Do we need to define these constants?

Reply:
I just thought it'd make what they are a bit more obvious.

Reply:
I mean, the attribute name and validates and format kind of already tell me everything I need to know - but maybe that's just me

Reply:
Seen as you both feel this is better I'll change it!